batchflow/models/tf/layers/pyramid.py

```diff
@@ -115,7 +176,7 @@ def aspp(inputs, layout='cna', filters=None, kernel_size=3, rates=(6, 12, 18), i
 
         for level in rates:
             x = conv_block(inputs, layout, filters=filters, kernel_size=kernel_size, dilation_rate=level,
-                           name='conv-%d' % level, **kwargs)
+                           name='atrous_conv-%d' % level, **kwargs)
```

Comment:
Why is `conv` not OK?

Reply:
Was using it to differentiate between ASPP layer convolutions and others during debugging. Removed